app/src/main/java/com/keylesspalace/tusky/entity/MastoList.kt

```diff
@@ -22,5 +22,8 @@ package com.keylesspalace.tusky.entity
 
 data class MastoList(
     val id: String,
-    val title: String
-)
+    val title: String,
+    val exclusive: Boolean?,
+) {
+    override fun toString(): String = title
```

Comment:
Is this needed? I wouldn't override toString of a data class if it can be avoided

Reply:
Yeah, I think the problem that this fixes was fixed in https://github.com/tuskyapp/Tusky/pull/3823/files